libass/ass_fontselect.c

```diff
@@ -389,13 +385,12 @@ ass_font_provider_add_font(ASS_FontProvider *provider,
                            int index, void *data)
 {
     int i;
-    int weight, slant;
     ASS_FontSelector *selector = provider->parent;
     ASS_FontInfo *info = NULL;
     ASS_FontProviderMetaData implicit_meta = {0};
+    FT_Face face;
 
     if (!meta->n_family) {
-        FT_Face face;
```

Comment:
afaict `face` isn’t actually used outside of this `if` block (anymore)

Reply:
Reverted.